cosmwasm/enclaves/shared/block-verifier/src/submit_block_signatures.rs

```diff
@@ -166,7 +149,6 @@ fn validate_inputs(
 ) -> Result<(), sgx_status_t> {
     validate_input_length!(in_header_len, "header", MAX_VARIABLE_LENGTH);
     validate_input_length!(in_commit_len, "commit", MAX_VARIABLE_LENGTH);
-    validate_input_length!(in_txs_len, "txs", MAX_TXS_LENGTH);
```

Comment:
Now all transactions (txs) are stored in the light client. Since a block can be at most 21 MiB, this shouldn't be an issue. It also simplifies the code because you don't have to filter what messages are stored and keep that code updated when you add new message types.

Reply:
5d80ceb89